TESTS/events/queue/main.cpp

```diff
@@ -357,3 +357,4 @@ int main()
     return !Harness::run(specification);
 }
 
+#endif // !DEVICE_USTICKER
```

Comment:
Missing newlines at the end of files. (Probably all of them).

Some tools or compilers complain about this.